bin/anthology/utils.py

```diff
@@ -182,8 +182,7 @@ def emit(self, record):
 def clean_whitespace(text, strip='left'):
     old_text = text
     if text is not None:
-        text = text.replace('\n', '')
-        text = re.sub(r'\s+', ' ', text)
+        text = re.sub(r' +', ' ', text)
         if strip == 'left' or strip == 'both':
```

Comment:
This old version would remove paragraph breaks from abstracts, which we don't want.